loudness/theme.json

```diff
@@ -9,20 +9,12 @@
 			"title": "Blank"
 		},
 		{
-			"name": "header-footer-only",
+			"name": "landing-page",
 			"postTypes": [
 				"page",
 				"post"
```

Comment:
Should we remove 'post' from this list? It's probably unlikely to be used on a post..

Reply:
I was wondering the same. :)
I shall do.